### PR TITLE
disabled field indices that are not necessary

### PR DESCRIPTION
**Pull request by averywpx**
The following fields are disabled: 
`'aIndex', 'alt', 'AN', 'codingGeneIds', 'contig', 'docId', 'domains', 'end', 'geno2mp', 'genotypes', 'gnomad_exome_coverage', 'gnomad_genome_coverage', 'mainTranscript', 'originalAltAlleles', 'pos', 'ref', 'rg37_locus', 'sortedTranscriptConsequences', 'start', 'transcriptIds', 'wasSplit', 'xstart', 'xstop'`

### CODE REVIEW

**Comment by hanars** [comment]
This looks good to me! This is ready for @mike-w-wilson to take a look at, we should get his approval before merging

**Review by mike-w-wilson — APPROVED** [review]
This looks good to me! I really like how you implemented the disable in the decorator!

**Pull request merged**
